src/KeyworkerManagement/index.js

```diff
@@ -28,6 +28,15 @@ class HomePage extends Component {
               <div className="padding-right-large">Check a prisoner's key worker and allocate manually.</div>
             </div>
           </div>
+          {this.props.user && (this.props.user.maintainAccess /* || this.props.user.migration */) &&
```

Comment:
hiding the admin tasks if the roles aren't there.  Migration role will be coming in the next story